docs/react/your-first-app/7-live-reload.md

```diff
@@ -1,120 +1,162 @@
+// Same old code from before.
+// change: Add React import.
+import { useState } from 'react';
+// CHANGE: Add `UserPhoto` type import.
+import type { UserPhoto } from '../hooks/usePhotoGallery';
+
+const Tab2: React.FC = () => {
+  // CHANGE: Add `deletePhoto()` method.
+  const { photos, addNewToGallery, deletePhoto } = usePhotoGallery();
+  // CHANGE: Add state for the photo to delete.
+  const [photoToDelete, setPhotoToDelete] = useState<UserPhoto>();
+
+  return (
+    <IonPage>
+      <IonContent>
+        {/* Same old code from before. */}
+
+        <IonFab vertical="bottom" horizontal="center" slot="fixed">
+          <IonFabButton onClick={() => addNewToGallery()}>
+            <IonIcon icon={camera}></IonIcon>
+          </IonFabButton>
+        </IonFab>
+
+        {/* CHANGE: Add action sheet for deleting photos. */}
+        <IonActionSheet
+          isOpen={!!photoToDelete}
+          buttons={[
+            {
+              text: 'Delete',
+              role: 'destructive',
+              icon: trash,
+              handler: () => {
+                if (photoToDelete) {
+                  deletePhoto(photoToDelete);
+                  setPhotoToDelete(undefined);
+                }
+              },
+            },
+            {
+              text: 'Cancel',
+              icon: close,
+              role: 'cancel',
+              handler: () => {
+                // Nothing to do, action sheet is automatically closed
+              },
+            },
+          ]}
+          onDidDismiss={() => setPhotoToDelete(undefined)}
+        ></IonActionSheet>
+      </IonContent>
+    </IonPage>
+  );
 };
 ```
 
-The selected photo is removed from the Photos array first. Then, we use the Capacitor Preferences API to update the cached version of the Photos array. Finally, we delete the actual photo file itself using the Filesystem API.
-
-Make sure to return the `deletePhoto` function so it is as a part of the hook API that we expose:
+Add a click handler to the `<IonImg>` element. When the app user taps on a photo in our gallery, we’ll display an [Action Sheet](../../api/action-sheet.md) dialog with the option to either delete the selected photo or cancel (close) the dialog.
 
 ```tsx
-return {
-  photos,
-  takePhoto,
-  deletePhoto,
-};
+<IonGrid>
+  <IonRow>
+    {photos.map((photo) => (
+      <IonCol size="6" key={photo.filepath}>
+        {/* CHANGE: Add a click event listener to each image. */}
+        <IonImg src={photo.webviewPath} onClick={() => setPhotoToDelete(photo)} />
+      </IonCol>
+    ))}
+  </IonRow>
+</IonGrid>
 ```
 
-Save this file, then tap on a photo again and choose the “Delete” option. This time, the photo is deleted! Implemented much faster using Live Reload. 💪
+Remember that removing the photo from the `photos` array triggers the `cachePhotos` method for us automatically.
+
+Tap on a photo again and choose the “Delete” option. The photo is deleted! Implemented much faster using Live Reload. 💪
```

Comment:
Missing the note here like Angular:

<img width="1470" height="626" alt="Image" src="https://github.com/user-attachments/assets/c44a0da2-dd76-4123-8f58-83a83f151604" />

Reply:
https://github.com/ionic-team/ionic-docs/pull/4201/commits/2722fc63b4c772ee73a7fefeee8ef8e02e78bdde